src/main/resources/db/migration/0250/V280__DocumentProducerFilesModifications.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE files ADD COLUMN file_date DATE;
+ALTER TABLE files ADD COLUMN caption TEXT;
+ALTER TABLE files ADD COLUMN width INTEGER;
+ALTER TABLE files ADD COLUMN height INTEGER;
```

Comment:
The `caption` column on `report_photos` should probably go away (with the values moved to this new column).

Reply:
Is it OK to remove that in a subsequent PR? The intention with this PR is to make as few changes to the original code as possible. While this particular file is different than what is in `pdd-backend`, I fear making that change will require other code changes which are out of scope of this PR.

Reply:
Yes, sorry, didn't mean to imply it should be part of this PR, just something that occurred to me as I was looking at this.

Reply:
Got it, I'll make sure to add it to the subsequent PR